src/hook/useNion.js

```diff
@@ -176,7 +179,7 @@ function useNion(declaration, deps = EMPTY_DEPS) {
 
     const props = [obj, actions, request, extra]
 
-    if (isDevtoolEnabled()) {
+    if (isDevtoolEnabled() && SUPPORTS_CAPTURE_STACK_TRACE) {
```

Comment:
Just double checking that we want to put this check here, instead of wrapping `Error.captureStackTrace` with it. Looks like `Error.prepareStackTrace` doesn't exist in ff too, would we still want to return `withStats` in ff?

Reply:
> Just double checking that we want to put this check here, instead of wrapping Error.captureStackTrace

I was not sure if skipping the `prepareStackTrace` step was going to cause issues with this dev tool. For now I think the safest is to just disable it.

> Looks like Error.prepareStackTrace doesn't exist in ff too

It's not standard. We are patching it: https://github.com/Patreon/nion/blob/8a8f99a4176c32b7bb16763897d3cb8084efe5e4/src/hook/useNion.js#L188